recipes/carna/meta.yaml

```diff
@@ -0,0 +1,34 @@
+package:
+  name: carna
+  version: "1.3.1"
+
+about:
+  home: http://www.bioinf.uni-freiburg.de/Software/Carna/
+  license: GPL
+  license_file: COPYING
+  summary: Multiple alignment of RNAs
+
+build:
+  number: 0
+  skip: True # [osx]
+
+source:
+  fn: carna-1.3.1.tar.gz
+  url: "http://www.bioinf.uni-leipzig.de/~will/Software/CARNA/carna-1.3.1.tar.gz"
+  sha256: 7c78463542c48322e2770eff544b46cffe5e4ce67f5fc58c05b4ead6063b541a
+
+requirements:
+  build:
+    - gecode >=4.4.0
+    - viennarna >=2.2.5
+    - locarna >=1.8.9
+    - gcc # [linux]
+    - libgcc # [linux]
```

Comment:
Feel free to create a PR for all of those. Thanks!
Otherwise I will look at it next week!